### DISCUSSION TITLE
Strawberry Music Player

### DISCUSSION THREAD

**Discussion by 5moufl** [elrumo/macOS_Big_Sur_icons_replacements#1359] [Icon request] [answered]
Hello,

could someone more talented than me create an icon for [strawberry](strawberrymusicplayer.org/)?

![241651795_296706328924932_4713275809597580257_n](https://user-images.githubusercontent.com/755140/152503550-5220437e-2e54-4874-bedb-e5f02e76d50b.png)

**Comment by bunkerdesign** [comment, marked as answer]
<img width="512" alt="Strawberry Music Player" src="https://user-images.githubusercontent.com/71460900/153430436-7820f48c-84fa-4d73-93a5-83eb8c930d81.png">
Maybe this will work
